test/core/end2end/flaky.bzl

```diff
@@ -24,6 +24,8 @@ FLAKY_TESTS = [
     "h2_compress_test@max_connection_age",
     "h2_compress_test@max_connection_idle",
     "h2_compress_test@retry_cancellation",
+    "h2_compress_test@retry_per_attempt_recv_timeout",
```

Comment:
The churn in this file suggests that we might want to change this list to identify flaky tests by the individual test, independent of the test fixture.  Most flakiness comes from the tests themselves, not the fixtures, so a test that is flaky in one fixture is very likely to also be flaky in other fixtures.  Identifying the flaky tests by both fixture and test will probably lead to a lot of churn based on which particular fixtures each test case has happened to flake on most recently, which is probably not a very useful signal.

Reply:
yeah... I'm going to keep an eye on it - I certainly think there's some improvements to make.